enterprise_scheduler/executor.py

```diff
@@ -85,30 +84,31 @@ def execute_task(self, task):
 
         ffdl_endpoint = task['endpoint']
         ffdl_authorization = task['user']
+        ffdl_authorization_pass = "temporary"
         ffdl_userinfo = task['userinfo']
         ffdl_zip = self._create_ffdl_zip(task)
         ffdl_manifest = self._create_manifest(task)
+        ffdl_ui_port = "32150"  ## FFDL UI hosting can vary
 
-        command = 'curl -X POST "' + ffdl_endpoint + '" ' \
-                  '-H "accept: application/json" ' \
-                  '-H "Authorization: ' + ffdl_authorization + '" ' \
-                  '-H "X-Watson-Userinfo: ' + ffdl_userinfo + '" ' \
-                  '-H "Content-Type: multipart/form-data" ' \
-                  '-F "model_definition=@' + ffdl_zip + ';type=application/zip" ' \
-                  '-F "manifest=@' + ffdl_manifest + '"'
+        task_headers = {"X-Watson-Userinfo": ffdl_userinfo}
 
-        #print('>>> Submitting \n {}'.format(command))
+        task_files = {'model_definition': open(ffdl_zip, 'rb'),
```

Comment:
should the files be opened into a variable, and closed at a finally clause similar to what we had for stderr/stdout when we were using popen ?